observability/otel/otel-collector/otel-collector-processors.adoc

```diff
@@ -427,24 +427,26 @@ include::snippets/technology-preview.adoc[]
 [source,yaml]
 ----
 # ...
-  config:
-    processors:
-      cumulativetodelta:
-        include: # <1>
-          match_type: strict # <2>
-          metrics: # <3>
-          - <metric_1_name>
-          - <metric_2_name>
-        exclude: # <4>
-          match_type: regexp
-          metrics:
-          - "<regular_expression_for_metric_names>"
-# ...
-----
-<1> Optional: Configures which metrics to include. When omitted, all metrics, except for those listed in the `exclude` field, are converted to delta metrics.
-<2> Defines a value provided in the `metrics` field as a `strict` exact match or `regexp` regular expression.
-<3> Lists the metric names, which are exact matches or matches for regular expressions, of the metrics to be converted to delta metrics. If a metric matches both the `include` and `exclude` filters, the `exclude` filter takes precedence.
-<4> Optional: Configures which metrics to exclude. When omitted, no metrics are excluded from conversion to delta metrics.
+mode: sidecar # <1>
+config: |
+  processors:
+    cumulativetodelta:
+      include: # <2>
+        match_type: strict # <3>
+        metrics: # <4>
+        - <metric_1_name>
+        - <metric_2_name>
+      exclude: # <5>
+        match_type: regexp
+        metrics:
+        - "<regular_expression_for_metric_names>"
+# ...
+----
+<1> This processor requires a stateful Collector deployment. So you can deploy it as a sidecar to the application that emits the delta temporality metrics.
```

Comment:
I am having a hard time understanding this callout. 

Are you saying:
If you want to deploy this processor as a sidecar, the processor requires a stateful collector deployment. You must deploy the sidecar to the application that emits the cumulative temporality metrics.

OR, are you saying:
This processor requires a stateful collector deployment and it must be deployed as a sidecar. You must deploy the sidecar to the application that emits the cumulative temporality metrics.

I also think you might be trying to say something different. TLDR: I think this needs clarification.